eng/common/scripts/tag-reviewer.ps1

```diff
@@ -0,0 +1,61 @@
+param(
+    [Parameter(Mandatory = $true)]
+    $RepoOwner,
+
+    [Parameter(Mandatory = $true)]
+    $RepoName,
+
+    [Parameter(Mandatory = $true)]
+    $GitHubUser,
+
+    [Parameter(Mandatory = $true)]
+    $PRNumber,
+  
+    [Parameter(Mandatory = $true)]
+    $AuthToken
+)
+
+if (-not $GitHubUser) {
+  Write-Host "No user provided for addition, exiting."
+  exit(0)
+}
+
+$headers = @{
+  Authorization = "bearer $AuthToken"
+}
+$uri = "https://api.github.com/repos/$RepoOwner/$RepoName/pulls/$PRNumber/requested_reviewers"
+
+try {
+  $resp = Invoke-RestMethod -Headers $headers $uri -MaximumRetryCount 3
+}
+catch {
+  Write-Error "Invoke-RestMethod [$uri] failed with exception:`n$_"
+  exit 1
+}
+
+# the response object takes this form: https://developer.github.com/v3/pulls/review_requests/#response-1
+# before we can push a new reviewer, we need to pull the simple Ids out of the complex objects that came back in the response
+$userReviewers = @($resp.users | % { return $_.login })
+$teamReviewers = @($resp.teams | % { return $_.slug })
+
+if (-not $userReviewers.Contains($GitHubUser)){
+  $userReviewers += $GitHubUser
```

Comment:
Just for clarity would this work for github teams as well?

Reply:
Yep!